### ISSUE TITLE
All devices not responding

### ISSUE THREAD

**Issue by korneliuscode** [homebridge/homebridge#3279]
### Current Situation

I set up homebridge and it worked the first day great in the UI and the Home app on iPhone. 
When I came back home the next day, it said „all devices were not responding“ on the iPhone. 

In Homebridge it still works but the Home app on iPhone  is not capable of doing anything. 

I rebooted the App, phone, homebridge and the raspberryPi - sill no change. 

How can I get this to work again or figure out where the link is broken between the two?

### Logs

```markdown
-
```

### Configuration

```json
-
```

### Environment

All current and standard on a Raspberry Pi. 

### Process Supervisor

hb-service

**Comment by pzipoy** [comment]
This [post](https://github.com/homebridge/homebridge/issues/3274#issuecomment-1331091924) might be worth checking out!

**Comment by dschrander** [comment]
This morning I had the same issue after upgrading to Homebridge version 1.6.0 yesterday.
Tried to change the mDNS Advertiser (as described in above [post](https://github.com/homebridge/homebridge/issues/3274#issuecomment-1331091924)), but this didn't help.

Removed the bridge from my home, did a reset of home and bridge, but then, when trying to add the reset bridge to my new home, Apple Home said it couldn't add the accessory because it is "out of compliance". 

I wasn't able to add the bridge anymore, until I switched back to version 1.5.1 and everything started working again.
